src/collections/_documentation/development/sdk-dev/envelopes.md

```diff
@@ -0,0 +1,448 @@
+  behavior should not be relied upon.
+
+**Envelope Headers:**
+
+`event_id`
+
+: **UUID String, required.** The identifier of the event or transaction.
+
+**Additional Item Headers:**
+
+`attachment_type`
+
+: *String, optional.* The special type of this attachment. Possible values are:
+
+  - **`event.attachment` (default)**: A standard attachment without special
+    meaning.
+  - `event.minidump`: A minidump file that creates an error event and is
+    symbolicated. The file should start with the `MDMP` magic bytes.
+  - `event.applecrashreport`: An Apple crash report file that creates an error
+    event and is symbolicated.
+  - `unreal.context`: An XML file containing UE4 crash meta data. During event
+    ingestion, event contexts and extra fields are extracted from this file.
+  - `unreal.logs`: A plain-text log file obtained from UE4 crashes. During
+    event ingestion, the last logs are extracted into event breadcrumbs.
+
+`content_type`
+
+: *String, optional.* The content type of the attachment payload. Defaults to
+  `application/octet-stream`.
+
+`filename`
+
+: *String, optional.* The name of the uploaded file without a path component.
+
+### Session
+
+Item type `"session"`. This Item contains a single session initialization or
+update to an existing session for Release Health.
+
+**Constraints:**
+
+- This item may occur multiple times per Envelope.
+- Ingestion may limit the maximum number of items per Envelope, see *Ingestion*.
+
+**Envelope Headers:**
+
+`sent_at`
+
+: *String, recommended.* The timestamp when the event was sent from the SDK as
+  [RFC 3339](https://tools.ietf.org/html/rfc3339) string. Used for clock drift
+  correction of the event timestamp. Timezone must be UTC.
+
+**Additional Item Headers:**
+
+*None*
+
+### UserReport
+
+Item type `"user_report"`. This item contains a user report JSON payload.
+
+**Constraints:**
+
+- This item may occur once per Envelope.
+- Attachments can be ingested separately from their events. It is recommended to
+  send them in the same Envelope.
+
+**Envelope Headers:**
+
+`event_id`
+
+: **UUID String, required.** The identifier of the event or transaction.
+
+**Additional Item Headers:**
+
+*None*
+
+### Reserved Types
+
+Reserved types may not be written by any implementation. They are reserved for
+future or internal use. This is the exhaustive list of reserved item types:
+
+- `security`
+- `unreal_report`
+- `form_data`
+
+## Ingestion
+
+This section describes how to ingest Envelopes into Relay or Sentry. The main
+ingestion endpoint for Envelopes is:
+
+```
+POST /api/<project_id>/envelope/
+```
+
+### HTTP Headers
+
+Envelope requests may contain all headers as regular store requests. The only
+accepted `content-type` is `application/x-sentry-envelope`, which is implied if
+it is missing.
+
+### Authentication
+
+In addition to regular header- and querystring authentication, the Envelope
+endpoint allows to authenticate with an Envelope header, instead. To choose
```

Comment:
This suggests you can authenticate the Envelope endpoint instead of the regular authentication, I think? I'm not clear on the use of "instead" here, since the previous sentence says "In addition to.."